pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -383,8 +383,9 @@ type MetaManager struct {
 }
 
 type MetaServer struct {
-	Enable bool `json:"enable"`
-	Debug  bool `json:"debug"`
+	Enable bool   `json:"enable"`
+	Debug  bool   `json:"debug"`
+	Server string `json:"server"`
```

Comment:
Do  we  need to spilit the server and port in config?

Reply:
No other place needs a port, it doesn’t have to be alone.